docs/EN_US/ECLStandardLibraryReference/SLR-Mods/RemoteDirectory.xml

```diff
@@ -4,17 +4,19 @@
 <sect1 id="RemoteDirectory">
   <title>RemoteDirectory</title>
 
-  <para><emphasis role="bold">STD.File.RemoteDirectory<indexterm>
+  <para><emphasis role="bold">STD.File.RemoteDirectory(</emphasis> <emphasis>
+  machineIP, dir </emphasis> <emphasis role="bold">[</emphasis> <emphasis>,
+  mask </emphasis> <emphasis role="bold">][</emphasis> <emphasis>, recurse
+  </emphasis> <emphasis role="bold">]</emphasis><emphasis
+  role="bold">[</emphasis> <emphasis>, planename</emphasis> <emphasis
```

Comment:
This one should be 'planeName' (capital N) because it should match with the 'machineIP'.